shard.yml

```diff
@@ -10,7 +10,7 @@ dependencies:
     version: ~> 0.4.0
   exception_page:
     github: crystal-loot/exception_page
-    version: ~> 0.3.0
+    version: ~> 0.4.0
```

Comment:
**chore:** This dependency update seems unrelated to adding a date header, so it should be a separate PR.
```suggestion
    version: ~> 0.3.0
```